vllm/sequence.py

```diff
@@ -681,6 +681,7 @@ def __init__(
     ) -> None:
         self.request_id = request_id
         self.seqs = seqs
```

Comment:
Should we assert `len(seqs)==1` after this PR?

Reply:
the assembled group of parallel sample still holds `len(seqs) > 1` . we can assert after further cleanup.